kivy/uix/camera.py

```diff
@@ -78,6 +78,14 @@ class Camera(Image):
     :attr:`resolution` is a :class:`~kivy.properties.ListProperty` and defaults
     to [-1, -1].
     '''
+    
+    apiID = NumericProperty(0)
+    '''Preferred video capture API ID to use when invoking the camera. If you are using
+    0, the API will be the default one::
```

Comment:
Should we mention that only specific core providers are supporting that?
How about using a `StringProperty`with default to `default` instead?

Reply:
I'm afraid I don't understand OpenCV well enough to understand your question.  Is it connected to different API's only being available on specific platforms?

I believe a NumericProperty would be better as the cv2.VideoCapture() function expects an integer for the api Preference.  And that way users can simply use cv.CAP_DSHOW, cv.CAP_MSMF, cv.CAP_V4L2, etc when attempting to call out a specific api.  If you still believe a StringProperty would be better, I defer to your judgement.  However, I'm not sure how to properly implement the conversion of a StringProperty back to a integer for cv2.VideoCapture().

Reply:
Thank you for the explanation.  I have attempted to make the changes that you requested.